packages/lyra/tests/lyra.test.ts

```diff
@@ -195,6 +195,51 @@ describe("lyra", () => {
     expect(searchResult.hits[0].id).toBe(id2);
   });
 
+  it("Shouldn't returns deleted documents", async () => {
+    const db = new Lyra({
+      schema: {
+        txt: "string",
+      },
+      stemming: false,
+    });
+
+    await db.insert({ txt: "stelle" });
+    await db.insert({ txt: "stellle" });
+    await db.insert({ txt: "scelte" });
+
+    const search = await db.search({ term: "stelle" });
+
+    const id = search.hits[0].id;
+
+    await db.delete(id);
+
+    const serach2 = await db.search({ term: "stelle" });
+
+    expect(serach2.count).toBe(1);
+  });
+
+  it("Shouldn't affects other document when deleted one", async () => {
+    const db = new Lyra({
+      schema: {
+        txt: "string",
+      },
+      stemming: false,
+    });
+
+    await db.insert({ txt: "abc" });
+    await db.insert({ txt: "abc" });
+    await db.insert({ txt: "abcd" });
+
+    const search = await db.search({ term: "abc", exact: true });
+
+    const id = search.hits[0].id;
+    await db.delete(id);
+
+    const search2 = await db.search({ term: "abc" });
```

Comment:
This line should also use `exact: true`, otherwise the tests passes even if #40 is not fixed.

Reply:
If you search for `abc` as an exact term, the result will be **0**, 'cause the only document available is the document that contains `txt: "abcd"`. 

![image](https://user-images.githubusercontent.com/11861080/180443759-d4670831-28f6-4a2c-aed0-045b347eecaf.png)

This expected result could be more realistic, but in both cases, it works correctly

Reply:
But it shouldn't be 0, that's the bug in #40. It should be 1, because there is a second document containing `"abc"` that should **not** be deleted.

Reply:
Got the point. The issue should be opened again.